### PR TITLE
[ENH] Add admonitions to task and beh pages

### PR DESCRIPTION
**Pull request by Remi-Gau**
- Blocked by https://github.com/bids-standard/bids-examples/pull/431
---

- add some admonitions to task and behavioral pages
- implement semantic line break in a few places
- change some wording / capitalization (e.g should --> SHOULD)

### CODE REVIEW

File: src/modality-specific-files/behavioral-experiments.md [hunk 1]
```diff
@@ -9,17 +15,20 @@ and a guide for using macros can be found at
...
 physiological (`_physio.tsv.gz`, `_physio.json`)
 and other continuous recordings (`_stim.tsv.gz`, `_stim.json`)
 as for tasks performed during MRI, electrophysiological or other neural recordings.
-Additionally, events files that do not include the mandatory `onset` and
-`duration` columns can still be included, but should be labeled `_beh.tsv`
-rather than `_events.tsv`.
+Additionally, events files
+that do not include the mandatory `onset` and `duration` columns
+MAY be included,
+but SHOULD be labeled `_beh.tsv` rather than `_events.tsv`.
```
Comment (Remi-Gau) [inline src/modality-specific-files/behavioral-experiments.md:31]: Shouldn't that be a MUST?

If a file does not have onset, duration then it is an invalid events file.

File: src/modality-specific-files/behavioral-experiments.md [hunk 2]
```diff
@@ -9,17 +15,20 @@ and a guide for using macros can be found at
...
-In addition to logs from behavioral experiments performed alongside imaging data
-acquisitions, one can also include data from experiments performed with no neural
-recordings.
-The results of those experiments can be stored in the `beh` directory using the same
-formats for event timing (`_events.tsv`), metadata (`_events.json`),
+In addition to logs from behavioral experiments
+performed alongside imaging data acquisitions,
+one MAY also include data from experiments
+performed with no neural recordings.
+The results of those experiments MAY be stored in the `beh` directory
```
Comment (Remi-Gau) [inline src/modality-specific-files/behavioral-experiments.md:22]: switched to using MAY here

File: src/modality-specific-files/behavioral-experiments.md (hunk 1 above)
Comment (effigies) [reply src/modality-specific-files/behavioral-experiments.md:31]: Yes, let's use a MUST.

File: src/modality-specific-files/behavioral-experiments.md [hunk 3]
```diff
@@ -30,7 +39,7 @@ In addition to the metadata that is either:
 -   REQUIRED for some data that can be found in the `beh` directory
     (for example `SamplingFrequency` and `StartTime` for `*_<physio|stim>.tsv.gz` files),
 
-it is RECOMMENDED to add the following metadata to the JSON files of this directory.
+It is RECOMMENDED to add the following metadata to the JSON files of this directory.
```
Comment (effigies) [inline src/modality-specific-files/behavioral-experiments.md:42]: The "In addition to the metadata" sentence is never completed. This is the final clause. It should not be capitalized.

Comment (Remi-Gau) [reply src/modality-specific-files/behavioral-experiments.md:42]: ha yes!! 

thanks

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -257,6 +257,9 @@ The keyword for the heading must be one of the following:
 - example
 - quote
 
+Do note put [macros](#using-macros) in admonitions,
```
Comment (sappelhoff) [inline CONTRIBUTING.md:260]:
```suggestion
Do not put [macros](#using-macros) in admonitions,
```

**Review by sappelhoff — APPROVED** [review]

**Review by effigies — APPROVED** [review]

**Pull request merged**